velox/functions/prestosql/aggregates/AverageAggregate.cpp

```diff
@@ -142,7 +142,7 @@ void registerAverageAggregate(
           }
         }
       },
-      {false /*orderSensitive*/},
+      {false /*orderSensitive*/, false /*isCompanionFunction*/},
```

Comment:
It is strange to have an API that has both isCompanion and withCompanionFunctions